orders/views.py

```diff
@@ -1,13 +1,21 @@
 import json
-from unicodedata import name
 
-import jwt
-
-from django.http  import JsonResponse
-from django.views import View
+from django.http     import JsonResponse
+from django.views    import View
 
 from orders.models         import Cart
-from core.token_dacorators import token_decorator
+from core.token_decorators import token_decorator
+
+class CartView(View):
+    @token_decorator
+    def delete(self, requset, cart_id):
+        try:
+            Cart.objects.get(id = cart_id, user = requset.user).delete()
+
+            return JsonResponse({'message' : 'No Content'}, status = 200)
```

Comment:
No content에 대한 상태 코드는 204입니다.
상태 코드 다시 한 번 확인해 주세요.

```suggestion

            return JsonResponse({'message' : 'No Content'}, status = 204)
```